test/entity.spec.js

```diff
@@ -614,15 +554,10 @@ describe('JHipster generator for entity', () => {
                             'skip-ktlint-format': true
                         })
                         .withGenerators([
-                            [
-                                require('../generators/entity/index.js'), // eslint-disable-line global-require
-                                'jhipster-kotlin:entity',
-                                path.join(__dirname, '../generators/entity/index.js')
-                            ],
                             [
                                 require('../generators/entity-server/index.js'), // eslint-disable-line global-require
-                                'jhipster-kotlin:entity-server',
-                                path.join(__dirname, '../generators/entity-server/index.js')
+                                'generator-jhipster-kotlin:entity-server',
```

Comment:
I'm curious: was adding `generator-` prefix necessary? At work I needed to overrides entity-server, but without overriding also entity subgen it was not working